build-system/tasks/karma.conf.js

```diff
@@ -68,7 +69,14 @@ module.exports = {
     debug: true,
     basedir: __dirname + '/../../',
     transform: [
-      ['babelify', {'global': isTravisBuild(), 'sourceMapsAbsolute': true}],
+      [
+        'babelify',
+        {
+          'global': isTravisBuild(),
```

Comment:
Curious: Why Travis and not local? This is used for running tests in both cases.

Reply:
Not sure, I actually didn't change this line.

Reply:
Yeah, what I meant was now that you're relying on this to transform `node_modules`, it's necessary to enable `global: true` for local testing and on Travis.

Reply:
I think https://travis-ci.org/ampproject/amphtml/jobs/548325896 is failing because of this too.

Reply:
The link you posted confirms my suspicion. The failing runs are all coverage runs, which uses a different `babelify` config. I've sent out a fix in #22951, which I believe should handle this.